python/lsst/daf/butler/tests/cliCmdTestBase.py

```diff
@@ -103,11 +103,11 @@ def run_missing(self, inputs, expectedMsg):
             followed by arguments, option keys and option values.
         expectedMsg : `str`
             An error message that should be present in stdout after running the
-            subcommand.
+            subcommand. Can be a regular expression string.
         """
         result = self.run_command(inputs)
         self.assertNotEqual(result.exit_code, 0, clickResultMsg(result))
-        self.assertIn(expectedMsg, result.stdout)
+        self.assertRegex(result.stdout, expectedMsg)
```

Comment:
If these tests cause problems ever again I vote for dropping them.  I don't think I've ever met a stringification test that added net value, even for classes that actually had broken stringification.